automation/taskcluster/lib/tasks.py

```diff
@@ -423,21 +423,22 @@ def craft_raptor_signing_task(
         )
 
     def craft_release_signing_task(
-        self, build_task_id, apk_paths, channel, is_staging,
+        self, build_task_id, apk_paths, channel, is_staging, index_channel=None
```

Comment:
Nit: I'm not sure to follow why we need to create `index_channel` and we can't reuse `channel` alone. Could you explain it to me?

Reply:
Sure: we're currently deploying `nightly` to the `production` channel (just on a different track).
So, when we create the signing task, we tell it to use the `production` channel's key (`channel="production"`). However, when we create the indexes, the nightly app _isn't_ the production app, so we don't want to publish it on the production app's index. So, we need to override the index's channel variable.

So, for nightlies:
`craft_release_signing_task(..., channel="production", index_channel="nightly")`

Reply:
Discussed offline: I don't see any better solution. Thanks for the explanations!